bundles/org.openhab.binding.hdmicec/README.md

```diff
@@ -0,0 +1,67 @@
+# HdmiCec Binding
+
+Binding to control devices connected via HDMI using CEC protocol. For example if you have a Raspebrry Pi 3 connected to a AV Receiver or TV, you can control other devices connected that support CEC. This is ideal for Sony PlayStation 4 which can't be turned on via IR, or Amazon Fire TV (stick/tv/cube) devices that use BT remotes.
+
+## Supported Things
+
+It should support devices which support CEC. See [libcec supported devices](http://libcec.pulse-eight.com/Vendor/Support) for a list)
+
+## Discovery
+
+This version supports manual discovery. You can invoke discovery via the Refresh link in Paper UI. It will invoke device discovery on the CEC bus. This can take ~30s to switch on and enumerate the devices so is not suitable for continuous background refresh.
```

Comment:
Reference to Paper UI, which was replaced by main UI in openHAB 3.

Reply:
Good catch. I'll update.